src/pages/form-validation/list/index.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback, useState } from 'react';
+import {
+    AppBar,
+    Card,
+    Hidden,
+    IconButton,
+    InputProps,
+    Switch,
+    TextField,
+    Toolbar,
+    Typography,
+} from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import { makeStyles, Theme, useTheme } from '@material-ui/core/styles';
+import { useDispatch } from 'react-redux';
+import { TOGGLE_DRAWER } from '../../../redux/actions';
+import { InfoListItem } from '@pxblue/react-components';
+import List from '@material-ui/core/List';
+import ReportIcon from '@material-ui/icons/Report';
+import { Dns } from '@material-ui/icons';
+
+const useStyles = makeStyles((theme: Theme) => ({
+    containerWrapper: {
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'start',
+        flex: '1 1 0',
+    },
+    container: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        backgroundColor: 'white',
+        maxWidth: 800,
+        marginRight: theme.spacing(4),
+        marginLeft: theme.spacing(4),
+        marginTop: theme.spacing(4),
```

Comment:
Can we combine these (since there is nothing below it, we don't have to be super particular about leaving off the bottom)?

```suggestion
        margin: theme.spacing(4),
```

or
```
margin: `${theme.spacing(4)}px ${theme.spacing(4)}px 0px`,
```